frontend/src/descriptions/submitBookKeeping.jsx

```diff
@@ -0,0 +1,51 @@
+    "- **Transaktionslista** (hittas i internetbanken): '`<KOMMITÉNAMN+VERKSAMHETSÅR>_lp<LP>_transaktion.pdf`'\n" +
+    "- **Transaktionslista, Swish** (gäller endast styrIT, hittas i internetbanken): '`styrit<VERKSAMHETSÅR>_lp<LP>_swish.pdf`'\n\n" +
+
+    "Om du skulle behöva skicka flera filer under samma kategori så lägger du till perioden som dokumentet beskriver i slutet på namnet.  \n" +
+    "Ex: BarIT19 har ett verksamhetsår som börjar under styrIT18s verksamhetsår och slutar under styrIT19s verksamhetsår.\n" +
+    "Då behöver kassören skicka in två huvudböcker för att täcka in hela sitt verksamhetsår efter att styrIT19 gått på.\n" +
+    "Då kommer de heta  \n" +
+    "'`barit19_lp1_huvudbok_lp3lp4.pdf`'  \n" +
+    "och  \n" +
+    "'`barit19_lp1_huvudbok_lp1.pdf`'  \n" +
+    "i LP1.  \n" +
+    "&nbsp;  \n" +
+    "Ibland är det även relevant att lägga till huvudboken för hela sektionen, exempelvis ifall en kommité har bokfört en förändring mot an annan kommités bankkonto.\n" +
+    "Huvudboken för hela sektionen ska då heta  \n" +
+    "'`sektionen<STYRITS_VERKSAMHETSÅR>_lp<LP>_huvudbok.pdf`'.  \n" +
+    "Ex: BarIT19 har fått pengar från FooIT19 för att hålla i en sittning som FooIT bokför under LP1.\n" +
+    "När BarIT19 då lämnar in sin bokföring inför sektionsmötet LP3 ska även sektionens huvudbok skickas med under namnet  \n" +
+    "'`sektionen19_lp1_huvudbok.pdf`'  \n" +
+    "utöver de vanliga dokumenten.  \n" +
+    "&nbsp;  \n" +
+    "##### Bra att tänka på:\n" +
+    "- När granskningen sker ska fysisk bokföring (verifikat, kvitton, deltagarlistor, internfordringar, samt fakturor) finnas tillgängligt för revisorerna. Verifikaten bör även vara underskivna, detta är särskilt viktigt inför ansvarsbefrielse.\n" +
+    "- Om kassören av någon anledning ligger efter i bokföringen inför granskningen ska bokföring ändå skickas in till den grad det går samt så behöver revisorerna få veta att kassören ligger efter samt varför.\n" +
+    "- Om ni är osäkra på något, kolla i ekonomisk policy, stadgar, reglemente, eller fråga någon av revisorerna eller styrITs kassör.\n" +
+    "- Revisorerna är dina vänner, vi är här för att hjälpa dig med bokföringen!\n" +
+
+    "##### Bra att kolla på innan du lämnar in bokföringen:\n" +
+    "- Kolla så att bankkontot stämmer överens mellan transaktionslistan och huvudboken för hela sektionen då transaktioner kan bokföras på olika kostnadsställen och om de inte gör det lägg en kommentar på varför de inte gör det.\n" +
+    "- Kolla så att ni har följt ekonomisk policy gällande intern representation, extern representation, aspning, profilering, samt överlämning.\n" +
+    "  - Om ni fått dispens för någon av dessa se till att det finns en kommentar i verifikatet där ni refererar till styrelsemötet.\n" +
+    "- Om det finns deltagarlistor på tillfällen som det har bjudits på mat (ex teambuilding och aspning)";
```

Comment:
boi it should be "Att det finns ..."

Reply:
Whoops, something must have went wrong during the merge locally, 1 sec.